Configuration/Template.php

```diff
@@ -158,4 +165,22 @@ public function allowArray()
     {
         return false;
     }
+
+    /**
+     * @param array $owner
+     */
+    public function setOwner(array $owner)
```

Comment:
technically, this is not always an array (you even removed the condition ensuring that non-array callables are not handled anymore by the listener).